psd-web/app/controllers/account_name_controller.rb

```
@@ -0,0 +1,18 @@
+class AccountNameController < ApplicationController
+  def show
+    @user = current_user
+    render "account/name"
+  end
+
+  def update
+    @user = current_user
+
+    @user.name = params.dig(:user, :name)
```

I still think this is wrong this controller needs a `user[name]` params to work correctly. This now will fail silently. 
what's wrong with strong parameter that's been conceived for this purpose.

---

strong parameter is useful when you're bulk updating from params (eg `.update_attributes(params)`) and want to whitelist which ones are ok. As it's only one attribute here, I think it's ok to pluck it straight out the params hash (if present).